web/jsonapi/jsonapi.go

```diff
@@ -52,6 +52,39 @@ func Data(c *gin.Context, statusCode int, o Object, links *LinksList) {
 	c.Data(statusCode, ContentType, body)
 }
 
+// DataList can be called to send an multiple-value answer with a
+// JSON-API document contains multiple objects.
+func DataList(c *gin.Context, statusCode int, objs []Object) {
+	var err error
+	objsMarshaled := make([]json.RawMessage, len(objs))
+	for i, o := range objs {
+		var j json.RawMessage
+		j, err = MarshalObject(o)
```

Comment:
Why not remove the `var err error` at top of the func and `var j json.RawMessage`? And initialize them with:

```go
j, err := MarshalObject(o)
```